src/components/calcite-color-picker-hex-input/calcite-color-picker-hex-input.scss

```diff
@@ -2,27 +2,26 @@ $input-side-offset--small: 16px;
 $input-side-offset: 18px;
 
 :host {
-  display: block;
+  @apply block;
 }
 
 .container {
-  width: 100%;
+  @apply w-full inline-grid items-center;
   display: -ms-inline-grid;
```

Comment:
Remove IE hacks now that we've dropped support?

Reply:
I removed all `-ms` from the file